content/en/configuration/coap-gateway.md

```diff
@@ -0,0 +1,154 @@
+| ---------- | -------- | -------------- | ------- |
+| `log.debug` | bool | `Set to true if you would like to see extra information in logs.` | `false` |
+| `log.dumpCoapMessages` | bool | `Set to true if you would like to dump raw messages.` | `false` |
+
+### CoAP API
+
+CoAP API as specified in the [Open Connectivity Foundation - Device to Cloud Services](https://openconnectivity.org/specs/OCF_Device_To_Cloud_Services_Specification_v2.2.3.pdf).
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `api.coap.address` | string | `Listen specification <host>:<port> for coap client connection.` | `"0.0.0.0:5684"` |
+| `api.coap.externalAddress` | string | `External address including public domain/IP for coap client connection.` | `"coap-gw.example.com:5684"` |
+| `api.coap.maxMessageSize` | int | `Max message size which can be send/receive via coap. i.e. 256*1024 = 262144 bytes.` | `262144` |
+| `api.coap.ownerCacheExpiration` | string | `Time limit of how long to keep subscribed to device updates after last use of the given cache item.` | `1m` |
+| `api.coap.subscriptionBufferSize` | int | `The maximum buffer size for one events subscription.` | `1000` |
+| `api.coap.goroutineSocketHeartbeat` | string | `Interval time to check live service.` | `4s` |
+| `api.coap.keepAlive.timeout` | string | `Time limit to close inactive connection.` | `20s` |
+| `api.coap.blockwiseTransfer.enabled` | bool | `If true, enable blockwise transfer of coap message.` | `false` |
+| `api.coap.blockwiseTransfer.blockSize` | int | `Size of blockwise transfer block.` | `1024` |
+| `api.coap.tls.enabled` | bool | `If true, require server certificate for ssl connection.` | `true` |
+| `api.coap.tls.caPool` | string | `File path to the root certificate in PEM format which might contain multiple certificates in a single file.` |  `""` |
+| `api.coap.tls.keyFile` | string | `File path to private key in PEM format.` | `""` |
+| `api.coap.tls.certFile` | string | `File path to certificate in PEM format.` | `""` |
+| `api.coap.tls.clientCertificateRequired` | bool | `If true, require client certificate.` | `true` |
+
+#### OAuth2.0 Client
+
+OAuth2.0 Client is used to issuing an authorization code used by the Onboarding Tool during the device provisioning which is then exchanged for an access token during the [SignUp operation](https://plgd.dev/architecture/component-overview/#cloud-registration).
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `api.coap.authorization.ownerClaim` | string | `Claim used to identify owner of the device.` | `"sub"` |
+| `api.coap.authorization.deviceIdClaim` | string | `Claim used to identify device id of the device. Empty means that JWT doesn't contain it.` | `""` |
+| `api.coap.authorization.providers[].Name` | string | `Provider name, the register request of the device must contain one of the names of providers.` | `""` |
```

Comment:
`Name` ??? right is `name`

Reply:
I just moved the file from cloud repo. Will change it to name.